src/components/DNSenumTool/DNSenumTool.tsx

```diff
@@ -1,4 +1,4 @@
-import { Button, LoadingOverlay, Stack, TextInput, Switch, Checkbox } from "@mantine/core";
+import { Button, LoadingOverlay, Stack, TextInput, Switch, Modal } from "@mantine/core";
```

Comment:
You also need to import checkAllCommandsAvailability and Installation Modal. 
```suggestion
import { Button, LoadingOverlay, Stack, TextInput, Switch, Modal } from "@mantine/core";
import { checkAllCommandsAvailability } from "../../utils/CommandAvailability";'
import InstallationModal from "../InstallationModal/InstallationModal";

```